code/__defines/math_arfs.dm

```diff
@@ -0,0 +1,3 @@
+//from bay
+#define ceil(x) (-round(-(x)))
```

Comment:
RE: Does the exact same thing as CEILING(x, 1)

Reply:
does it ho

Reply:
Going to keep Ceil since when I tried to change to ceiling DM fucking died 3 times

Reply:
It quite literally does.
![image](https://user-images.githubusercontent.com/44811257/88675134-3af24980-d0b0-11ea-81fd-a52208fecdf8.png)
As the DM errors I saw earlier on the discord, you failed to match the required inputs for the macro, which in this case, would be `CEILING(x, 1)`

Reply:
that's.. what I did though?